### ISSUE TITLE
[🐛 Bug]: IWebElement.GetDomProperty("style") returns "System.Collections.Generic.Dictionary`2[System.String,System.Object]" (as string)

### ISSUE THREAD

**Issue by Mek7** [SeleniumHQ/selenium#14846]
### What happened?

I am upgrading Selenium test code in .NET 8 from Selenium.Support version 4.25.0 to 4.27.0
Because IWebElement.GetAttribute() method is now marked as deprecated, I replaced it with GetDomProperty().
For example:
`webDriver.FindElement(By....whatever).GetDomProperty("style")`
should return a string like
`background: rgb(65, 131, 215); height: 8.125%;`
but returns this string:
`System.Collections.Generic.Dictionary`2[System.String,System.Object]`
Looks like a bug - that string is the string representation of calling .ToString() on a Dictionary instance.
Or is it supposed to work like this?

When I use GetDomAttribute() instead of GetDomProperty(), the correct string is returned:
`background: rgb(65, 131, 215); height: 8.125%;`

### How can we reproduce the issue?

```shell
See above in "What happened"
```

### Relevant log output

```shell
Not sure - using Selenium in Visual Studio unit test framework. Not seeing any useful stuff in debug output.
```

### Operating System

Windows 11

### Selenium version

.NET 4.27.0

### What are the browser(s) and version(s) where you see this issue?

Firefox 132.0.2

### What are the browser driver(s) and version(s) where you see this issue?

GeckoDriver 0.35.0

### Are you using Selenium Grid?

no

**Comment by nvborisenko** [comment]
For maintainers:

**GetDomProperty:**

```csharp
var style = element.GetDomProperty("style");
Console.WriteLine(style); // System.Object[]
```

```
21:00:42.948 TRACE HttpCommandExecutor: >> GET RequestUri: http://localhost:57238/session/92b41016a843a63333ee03f6aa03c648/element/f.533155AB6C15ADAF23E24748C23919EA.d.A26436E61238A5F5AA2A84DDBC427528.e.12/property/style, Content: null, Headers: 3
21:00:42.955 TRACE HttpCommandExecutor: << StatusCode: 200, ReasonPhrase: OK, Content: System.Net.Http.HttpConnectionResponseContent, Headers: 1
{"value":["height","line-height","width"]}
```

**GetDomAttribute:**

```csharp
var style = element.GetDomAttribute("style");
Console.WriteLine(style); // height:20px;line-height:20px;width:20px
```

```
21:11:53.355 TRACE HttpCommandExecutor: >> GET RequestUri: http://localhost:56622/session/e66e568e99e1ec12698fef989bd7173e/element/f.36FADB5E54CA2D7404527200702A6B67.d.BBE3AC63C943932979074E4D6C823EDA.e.13/attribute/style, Content: null, Headers: 3
21:11:53.362 TRACE HttpCommandExecutor: << StatusCode: 200, ReasonPhrase: OK, Content: System.Net.Http.HttpConnectionResponseContent, Headers: 1
{"value":"height:20px;line-height:20px;width:20px"}
```

I thought I broke it recently, but no, the same behavior using "old" selenium versions.

For @Mek7, seems in your case it is better to use DomAttribute?

In any case string representation of `GetDomProperty()` should be better! Given that it returns `string`, it must be even better!

**Comment by Mek7** [comment]
Yes, GetDomAttribute() works OK but GetDomProperty() returns the ToString() result of a dictionary.
Both methods return string. When I migrated the old code, when one did not work, I used the other :)

**Comment by ratoaq2** [comment]
It seems we have a similar issue with java.
`element.getDomProperty("style")` returns something like `[height, visibility]`
